core/blockchain.go

```diff
@@ -41,16 +41,17 @@ import (
 
 	"github.com/ava-labs/coreth/consensus"
 	"github.com/ava-labs/coreth/consensus/misc/eip4844"
-	"github.com/ava-labs/coreth/core/rawdb"
 	"github.com/ava-labs/coreth/core/state"
 	"github.com/ava-labs/coreth/core/state/snapshot"
 	"github.com/ava-labs/coreth/core/types"
 	"github.com/ava-labs/coreth/internal/version"
 	"github.com/ava-labs/coreth/params"
+	crawdb "github.com/ava-labs/coreth/plugin/evm/rawdb"
```

Comment:
not sure I like this `crawdb` name.

Reply:
Changed to `customrawdb` - alternatively we could call libevm/rawdb as ethrawdb, but it would produce more diffs in the code.